### PR TITLE
Add jquery-outside-events@1.1 with git auto-update

### PR DESCRIPTION
**Pull request by LeaYeh**
**git repo url:** https://github.com/cowboy/jquery-outside-events/tree/v1.1
**Watch 8
 Star 211
 Fork 32**
@maruilian11 could you help me check out the PR for #6203 ? thank you~
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity 
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/jquery-outside-events/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "jquery-outside-events",
+  "filename": "jquery.ba-outside-events.min.js",
+  "version": "1.1",
+  "homepage": "https://github.com/cowboy/jquery-outside-events",
+  "keywords": [
+    "jQuery",
+    "clickoutside"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-outside-events/package.json]: Is this `clickoutside` made by yourself?

There are 3 more *-outside plugin: 
https://github.com/cowboy/jquery-outside-events/blob/v1.1/jquery.ba-outside-events.js#L30-L33

I will choose to use outside only.

**Comment by LeaYeh** [comment]
i just try to find key word from readme, but i think ouside is better.

**Comment by maruilian11** [comment]
@LeaYeh i think you can add `description`.

**Comment by LeaYeh** [comment]
@maruilian11  i add description field and rebase the commit, could you check this pr again?

**Comment by maruilian11** [comment]
@LeaYeh i think this PR is ok, could you rebase on our latest master branch? thanks!

**Pull request merged**